pyam/core.py

```diff
@@ -766,7 +766,7 @@ def rename(self, mapping=None, inplace=False, append=False,
 
     def convert_unit(self, current, to=None, factor=None, registry=None,
                      context=None, inplace=False):
-        """Convert all data having *current* units to new units.
+        r"""Convert all data having *current* units to new units.
```

Comment:
is the `r` on purpose?

Reply:
Yes: I built the docs, and saw that `'gwp_'` further down was being interpreted as a ReST link (trailing underscore) when I didn't intend that. So, I added `\` to escape the underscore; but then my Python linter was complaining that that was a special character. `r` makes it a ["raw string"](https://docs.python.org/3/reference/lexical_analysis.html#index-19) in which `\` is a literal character.